InstructionsView.py

```diff
@@ -19,25 +19,26 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 
-import gi
-gi.require_version('Gtk', '3.0')
-from gi.repository import Gtk, GObject, Pango, GdkPixbuf
-
 import os
 from glob import glob
 
 from Globales import COLORES
 from JAMediaImagenes.ImagePlayer import ImagePlayer
 
 
+import gi
+from gi.repository import Gtk, GObject, Pango, GdkPixbuf
+gi.require_version('Gtk', '3.0')
+
+
 class HelpSlideShow(Gtk.EventBox):
 
     def __init__(self):
 
         Gtk.EventBox.__init__(self)
 
         self.modify_bg(Gtk.StateFlags.NORMAL, COLORES["toolbar"])
-        #self.set_border_width(20)
+        # self.set_border_width(20)
```

Comment:
As a reviewed, I also accept removing commented out lines rather than fix the comment indentation.

Reply:
s/reviewed/reviewer